### PR TITLE
Introduce the household schema and accommodation section for NI

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
Adding the Household schema and accommodation section for Northern Ireland - [See Trello](https://trello.com/c/HqThoMil/3194-introduce-hh-schema-build-accommodation-ni-s)

### How to review 
[Link to the overflow diagram ](https://overflow.io/s/30TRR3/?node=cff3e722)
Check the jsonnet all makes sense
Check the rendered schema matches the prototype / changes in trello.
Make sure routing and validation work as specified.

[Quick Launch](https://v3-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-NIR&url=https://gist.githubusercontent.com/petechd/695f39b9dce2d0a8ee865063ebef32ec/raw/98672a35850804eb5600cf6fb2c1331e276df538/census_household_gb_nir.json)

### CODE REVIEW

**Comment by berroar** [comment]
[Introduce HH Schema & Build Accommodation NI (S)](https://trello.com/c/HqThoMil/3194-introduce-hh-schema-build-accommodation-ni-s)

File: data-source/jsonnet/northern-ireland/blocks/household/accommodation/adapted.jsonnet [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+{
+  type: 'Question',
+  id: 'adapted',
+  question: {
+    id: 'adapted-question',
+    mandatory: false,
+    type: 'MutuallyExclusive',
+    title: {
+      text: 'Has <em>{address}</em>  been designed or adapted for any of the following?',
```
Comment (MebinAbraham) [inline data-source/jsonnet/northern-ireland/blocks/household/accommodation/adapted.jsonnet]:
```suggestion
      text: 'Has <em>{address}</em> been designed or adapted for any of the following?',
```

File: data-source/jsonnet/northern-ireland/blocks/household/accommodation/type_of_flat.jsonnet [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        {
+          label: 'In a commercial building',
+          value: 'In a commercial building',
+          description: 'For example, in an office building, hotel, or over a shop',
+        },
+      ],
+      type: 'Radio',
+    }],
+  },
+  routing_rules: [{
```
Comment (MebinAbraham) [inline data-source/jsonnet/northern-ireland/blocks/household/accommodation/type_of_flat.jsonnet]: Technically this routing rule is redundant as it goes to `adapted` next anyway.

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Looks good, minor comments.

**Review by MebinAbraham — COMMENTED** [review]
Central heating question answer label is: 'Select all that apply'
Should be: 'Select all that apply, whether or not you use it'

**Review by MebinAbraham — COMMENTED** [review]
On overflow, the 'Select all that apply' has a colon so (Select all that apply:). Should check the importance of this with @donnap573

**Comment by berroar** [comment]
> Central heating question answer label is: 'Select all that apply'
> Should be: 'Select all that apply, whether or not you use it'

This was wrong on the Prototype/Overflow diagram, we checked with @donnap573  and she asked us to copy the guidance from Central heating ENG/WLS and use 'Select all that apply' here.

**Review by MebinAbraham — APPROVED** [review]

**Comment by berroar** [comment]
Updated wording on the 'renewables' question as requested by @donnap573

File: data-source/jsonnet/northern-ireland/blocks/household/accommodation/accommodation_type.jsonnet [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    {
+      goto: {
+        block: 'type-of-flat',
+        when: [rules.accommodationIsFlat],
+      },
+    },
+    {
+      goto: {
+        block: 'adapted',
```
Comment (JamesGardiner) [inline data-source/jsonnet/northern-ireland/blocks/household/accommodation/accommodation_type.jsonnet:52]: Should this be the same as the eng & wal or different, as it currently different for the bare goto

Comment (JamesGardiner) [reply data-source/jsonnet/northern-ireland/blocks/household/accommodation/accommodation_type.jsonnet:52]: The goto in the England & Wales schema is 'self-contained', just want to check this is a known difference

Comment (JamesGardiner) [reply data-source/jsonnet/northern-ireland/blocks/household/accommodation/accommodation_type.jsonnet:52]: https://github.com/ONSdigital/eq-survey-runner/blob/29bfb158cb027aa6be209ca484338ff2673c5a74/data-source/jsonnet/england-wales/blocks/household/accommodation/accommodation_type.jsonnet#L52

Comment (berroar) [reply data-source/jsonnet/northern-ireland/blocks/household/accommodation/accommodation_type.jsonnet:52]: Yeah this is supposed to have changed, the 'self-contained' question has been removed completely, and the 'adapted' question is a new one for NI.

**Review by JamesGardiner — APPROVED** [review]

File: app/translations/census_household_gb_nir.pot [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+"Answer for: What type of renewable energy systems does <em>{address}</em>"
+" have?"
+msgid "Solar panels for heating water"
+msgstr ""
+
+#. answer-id: renewables-answer
+msgctxt ""
+"Answer for: What type of renewable energy systems does <em>{address}</em>"
+" have?"
+msgid "Wind Turbine"
```
Comment (petechd) [inline app/translations/census_household_gb_nir.pot]: Looks like it's still capital T in .pot file(only)

**Review by donnap573 — APPROVED** [review]

**Pull request merged**